docs/java/features/rest/clients/scp-workflow-rest-api.md

```diff
@@ -0,0 +1,229 @@
+
+##### Identify necessary OAuth Scopes
+The REST API is protected and requires authenticating with an OAuth 2.0 access token. Each particular API endpoint requires the calling REST client to provide an access token valid for the respective endpoint. That is, the token must be issued for the respective OAuth scope that corresponds to the desired API endpoint.
+
+Let us figure out which OAuth scope is relevant for our application use case. We see on the [API documentation](https://api.sap.com/api/SAP_CP_Workflow_CF/resource) that the endpoint ``/v1/workflow-definitions`` is assigned to the scope `WORKFLOW_DEFINITION_GET`.
+
+##### Create Service Instance JSON Configuration
+Open a text editor of your choice and save a JSON file on your system with the following content:
+```
+{
+  "authorities": ["WORKFLOW_DEFINITION_GET"]
+}
+```
+Remember where you've saved the file, you'll need it later.
+
+##### Create Service Instance
+
+Open the command line and authenticate at your Cloud Foundry organization by invoking ``cf login``.
+
+Consider specifying the respective subaccount, organization and space with ``cf target`` if necessary.
+
+Use ``cd`` to navigate to the directory that contains the JSON file created beforehand.
+Being in that directory, create the service instance as follows:
+```
+cf create-service workflow standard my-workflow-service -c <path-to-json-file>
+```
+This command creates an instance of the SCP Workflow Service in the CF space that your CLI points to. More specifically, it uses the service plan "standard" and takes the JSON configuration into account. Note that we named the service instance ``my-workflow``. You can name it as you want, it is just important to remember the name as you need it in your deployment descriptor `manifest.yml` later on.
+
+Once the service instance creation is finished, you can see the service instance in your CF space under ``Services`` and `Service Instances` in the left-hand side menu.
+
+
+##### Bind your App to Service Instance
+Open the file ``manifest.yml`` in your project and mention your service instance under `services`. Refer to this example:
+```
+applications:
+- name: awesome-app
+  memory: 1024M
+  timeout: 600
+  random-route: false
+  path: application/target/awesome-app-application.war
+  buildpacks:
+  - sap_java_buildpack
+  env:
+    TARGET_RUNTIME: tomee7
+    SET_LOGGING_LEVEL: '{ROOT: INFO, com.sap.cloud.sdk: INFO}'
+    JBP_CONFIG_SAPJVM_MEMORY_SIZES: metaspace:128m..
+  services:
+  - my-destination
+  - my-workflow
+  routes:
+  - route: <omitted-on-purpose>
+```
+Redeploy your app with ``cf push``.
+
+##### Take Note of API endpoint and OAuth Credentials
+After the deployment of the app happened, go to your CF space and navigate to ``Services`` and thereafter to `Service Instances`. You should see the service instance you created along with the information that is is bound to your application.
```

Comment:
I recommend going through the document with strg+f and replacing all the double quotes with single ones, so "`"  instead of "``" 

It looks to me like when you type you don't press space after hitting the "`" key. That is needed since how it is printed depends on the following character. Just FYI in case you don't know. Or is there a reason for the double ones?

Reply:
good hint, thanks